sklearn/mixture/bayesian_mixture.py

```diff
@@ -111,7 +118,14 @@ class BayesianGaussianMixture(BaseMixture):
         'kmeans' : responsibilities are initialized using kmeans.
         'random' : responsibilities are initialized randomly.
 
-    dirichlet_concentration_prior : float | None, optional.
+    weight_concentration_prior_type : {'dirichlet_process',
+            'dirichlet_distribution'}, defaults to 'full'.
```

Comment:
Two problems:

- This new line breaks the sphinx rendering in classes.rst. I am not sure how this should be fixed.
- The default value is `'dirichlet_process'`, not `'full'`.